include/opcode/riscv-opc.h

```diff
@@ -867,6 +934,7 @@
 #define CSR_TINFO 0x7a4
 #define CSR_TCONTROL 0x7a5
 #define CSR_MCONTEXT 0x7a8
+#define CSR_MNOISE 0x7a9
```

Comment:
Seems like you add new CRSs, but forgot to declare them by DECLARE_CSR.

Reply:
Besides, add new CSR should update the series of priv-reg* test cases.

Reply:
Thanks for the point out, I had added it.